src/cpuid-templates/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "cpuid-templates"
+version = "0.1.0"
+edition = "2018"
+license = "Apache-2.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+cpuid = { path="../cpuid" }
+
+[dev-dependencies]
+serde_json = "1.0.91"
```

Comment:
nit: github doesn't like the absence of the newline.

Reply:
Added.